index.html

```diff
@@ -82,6 +82,73 @@ <h1>I’m Azaria<br>
                 </ul>
             </div>
         </section>
+        <section class="about body flex-column bg-white w-90 b-rounded-right p-15">
+            <div class="title">
+                <h1>About<br> Myself</h1>
+            </div>
+            <div class="content">
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website 
+                    Look through some of my work and experience! If you like what you see and have a project
+                     you need coded, don’t hestiate to contact me.
+                </p>
+
+            </div>
+            <div class="connection">
+                <a href="#">LET’S CONNECT</a>
+            </div>
```

Comment:
- [ ] It would be better if we change the `<div class="connection">` to `<hx class="connection">`, this helps improves HTML semantics